Formula/sipp.rb

```diff
@@ -22,4 +21,8 @@ def install
     system "make", "DESTDIR=#{prefix}"
     bin.install "sipp"
   end
+
+  test do
+    assert_equal "SIPp", shell_output("#{bin}/sipp -v | grep SIPp | awk '{ print $1 }'").strip
```

Comment:
@MikeMcQuaid using your original recommendation:
```
Error: sipp: failed
<0> expected but was
<99>.
```

@DomT4 after appending `2>&1`:
```
Error: sipp: failed
<0> expected but was
<99>.
```

For reference, this is what `sipp -v` spits out:
```

 SIPp v3.5.1-TLS-PCAP-RTPSTREAM built Jun  2 2016, 12:31:50.

 This program is free software; you can redistribute it and/or
 modify it under the terms of the GNU General Public License as
 published by the Free Software Foundation; either version 2 of
 the License, or (at your option) any later version.

 This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.

 You should have received a copy of the GNU General Public
 License along with this program; if not, write to the
 Free Software Foundation, Inc.,
 59 Temple Place, Suite 330, Boston, MA  02111-1307 USA

 Author: see source files.

```
I was not successful in parsing out the lines of the version output using shell_output so the only code that I could make work without failing was the following (using @MikeMcQuaid's version variable recommendation):
```
    sipp_output = `#{bin}/sipp -v`.split(/$/)
    sipp_output.each { |o| print o }
    assert_match "SIPp v#{version}", sipp_output[0]
```
This prints out the `sipp -v` output and passes the test.